pages/index.tsx

```diff
@@ -54,18 +61,32 @@ export default function Home() {
         </div>
       </main>
 
+      <div>
+        <h1 className="text-center text-3xl font-bold">{NEW_USER}</h1>
+        <div className="flex flex-wrap w-full mx-auto justify-center">
+          {Array.from({ length: NUM_MEMBERS_NUMBER }).map((_, i) => (
+            <NewMembersCard
+              key={i}
+              newMemberFirstName={firstName}
+              newMemberLastName={lastName}
+              newMemberImageSrc={avatarImageSrc}
+            />
+          ))}
+        </div>
+      </div>
+
       <footer className={styles.footer}>
         <a
           href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
           target="_blank"
           rel="noopener noreferrer"
         >
-          Powered by{' '}
+          Powered by{" "}
```

Comment:
???

Reply:
automatically change due to mine vs code settings